pkgs/development/libraries/qt-6/modules/qtbase.nix

```diff
@@ -171,8 +172,9 @@ stdenv.mkDerivation rec {
   ] ++ lib.optionals (lib.meta.availableOn stdenv.hostPlatform libinput) [
     libinput
   ] ++ lib.optionals stdenv.hostPlatform.isDarwin [
-    apple-sdk_11
-    (darwinMinVersionHook "11.0")
+    # Per <https://doc.qt.io/qt-6/macos.html#supported-versions>.
+    apple-sdk_15
+    (darwinMinVersionHook "12.0")
```

Comment:
Now that I think about it, this hook isn’t needed. The propagated hook does the same thing. Maybe there should be a comment to that effect? (That the propagated SDK and hook are setting the minimum version even though Qt itself is building with a newer SDK version.)

Reply:
Yeah, makes sense, and I can do that. A possible alternative would be to define this stuff in a `let` at the top so that the target and build version are defined together and we can have only one comment linking to the place to look for updates; what do you think of that? It would also be more prominent so that we’re less likely to forget to bump it in future.

Reply:
That sounds like a good idea to define those all together.